docs/pages/unified/marketing-automation/api-reference/put-contact.mdx

```diff
@@ -9,16 +9,20 @@ curl --request POST \
   --url https://app.vessel.dev/api/unifications/marketing/contacts/update \
   --header 'Content-Type: application/json' \
   --header 'x-vessel-api-token: ' \
-  --header 'x-vessel-access-token: '\
+  --header 'x-vessel-access-token: '\ 
```

Comment:
ah was this extra space needed to get it to compile? I think my VS code automatically trims trailing spaces, so I might have to disable that

Reply:
Not sure, I think it's just automatically added by the generator